src/server/ua_server_config.c

```diff
@@ -89,6 +89,7 @@ UA_ServerConfig_clean(UA_ServerConfig *config) {
         config->logger.clear(config->logger.context);
     config->logger.log = NULL;
     config->logger.clear = NULL;
+    config->pLogger = NULL;
```

Comment:
Use `config->pLogger->clear` for the cleanup.
Because some users will not set the normal config->logger at all.

Reply:
Call pLogger->clear is pLogger is not NULL and pLogger != &config->logger. config->logger.clear must still be called as pLogger might be different from &config->logger. Is that right?

Reply:
I'd say yes if pLogger != logger and logger.clear != NULL.

Reply:
Added clear call and check